functors_task_1/student/include/impl.h

```diff
@@ -16,16 +25,22 @@ struct BeerOrganizer
  *
  * @note Only Corona and HoeGaarden are expensive
  */
-bool isExpensiveBeer(/**???*/)
+bool isExpensiveBeer(BeerBrand beer)
 {
+    if (beer == BeerBrand::HoeGaarden || beer == BeerBrand::Corona)
```

Comment:
usually you should avoid naked true and false in your code, please simplify it, returning the result of the expression